pubspec.yaml

```diff
@@ -0,0 +1,25 @@
+name: workmanager_tizen
+description: A new Flutter plugin project.
+version: 0.0.1
+homepage:
+
+environment:
+  sdk: '>=3.0.1 <4.0.0'
```

Comment:
Please change to `sdk: ">=2.18.0 <4.0.0"`.

Reply:
Fixed it!